core/src/test/scala/org/http4s/UrlFormSpec.scala

```diff
@@ -70,7 +66,7 @@ class UrlFormSpec extends Http4sSpec with ScalaCheck {
       UrlForm(Map("key" -> Seq("a", "b", "c"))).getFirstOrElse("notFound", "d") must_== "d"
     }
 
-    "construct consistently from kv-pairs or and Map[String, Seq[String]]" in check {
+    "construct consistently from kv-pairs or and Map[String, Seq[String]]" in prop {
```

Comment:
I don't know why, but this isn't compiling due to a diverging implicit expansion. Any thoughts?